kubevirt/k6t-only-instancetype/k6t-require-instancetype.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k6t-require-instancetype
+  annotations:
+    policies.kyverno.io/title: Require instanceType
+    policies.kyverno.io/category: KubeVirt
+    kyverno.io/kyverno-version: 1.6.2
+    policies.kyverno.io/minversion: 1.6.2
+    kyverno.io/kubernetes-version: "1.21-1.25"
```

Comment:
This annotation means you personally tested and verified its application on Kubernetes 1.21, 1.22, 1.23, 1.24, and 1.25. Was that the case? If not, please only specify the version(s) on which you actually tested it.